pkg/ccl/backupccl/backupinfo/backup_metadata.go

```diff
@@ -262,6 +262,14 @@ func writeDescsToMetadata(
 	return nil
 }
 
+func FileCmp(left backuppb.BackupManifest_File, right backuppb.BackupManifest_File) int {
+	if cmp := left.Span.Key.Compare(right.Span.Key); cmp != 0 {
+		return cmp
+	}
+
```

Comment:
I think it's a reasonable idea. Currently this only slightly affects the import span generation as it can possibly allow the `fileSpanStartAndEndKeyIterator` to yield finer grained keys in the case of ties in the start keys of files within a layer, but I don't think that changes things too much. The thing that makes me hesitant in making this change is that the code will lose compatibility with previously created manifest SSTs.

Reply:
yeah, wasn't suggesting we change it here, just curious.